### PR TITLE
[SC-563] Fix: Simplify ModuleFactory deployment

### PR DESCRIPTION
**Pull request by FHieser**

### CODE REVIEW

File: test/factories/ModuleFactory_v1.t.sol [hunk 1]
```diff
@@ -84,6 +88,64 @@ contract ModuleFactoryV1Test is Test {
...
+        factory.init(governanceContract, metadata, beacons);
+
+        IInverterBeacon_v1 currentBeacon;
+        for (uint i = 0; i < metadataSets; i++) {
+            (currentBeacon,) = factory.getBeaconAndId(metadata[i]);
+            assertEq(address(currentBeacon), address(beacons[i]));
+        }
+    }
+
+    function testInitFailsForUnevenArrays(uint number1, uint number2) public {
```
Comment (0xNuggan) [inline test/factories/ModuleFactory_v1.t.sol]: I'd rename this to "testInitFailsForMismatchedArrayLenghts". "Uneven" sounds like "it should revert if the array length is 3"

**Review by 0xNuggan — CHANGES_REQUESTED** [review]

**Comment by 0xNuggan** [comment]
1 thing, rest looks good to me.

**Review by marvinkruse — APPROVED** [review]
Makes the deployment a lot easier in regards to the MultiSig, so makes sense to me. Looks good to me!

**Comment by marvinkruse** [comment]
Just pushed a little fix, as the renamed test had a typo :D

**Pull request merged**